contentcuration/contentcuration/settings.py

```diff
@@ -25,9 +25,23 @@
 STORAGE_ROOT = "storage"
 DB_ROOT = "databases"
 
+static_files = os.path.join(BASE_DIR, 'contentcuration', 'static')
 STATIC_ROOT = os.getenv("STATICFILES_DIR") or os.path.join(BASE_DIR, "static")
 CSV_ROOT = "csvs"
 
+STATICFILES_DIRS = (
+    static_files,
```

Comment:
I'm not sure if this will work in production, since we need it to be configurable through an environment variable.

I think you want to have `static_files`'s value be either `STATICFILES_DIR` if defined, and have a fallback for dev environments.